discovery-provider/src/api/v1/users.py

```diff
@@ -837,26 +858,41 @@ def get(self, id):
 
 @full_ns.route("/content_node/<string:replica_type>", doc=False)
 class UsersByContentNode(Resource):
-    @full_ns.marshal_with(users_by_content_node_response)
-    # @cache(ttl_sec=30)
-    def get(self, replica_type):
-        """New route to call get_users_cnode with replica_type param (only consumed by content node)
-        - Leaving `/users/creator_node` above untouched for backwards-compatibility
-
+    @ns.doc(
+        id="""Get Users By Replica Type for Content Node""",
+        description="""
+        (Only consumed by Content Node) Gets users that have a given Content Node endpoint as
+        their primary, secondary, or either (depending on the replica_type passed).
         Response = array of objects of schema {
             user_id, wallet, primary, secondary1, secondary2, primarySpId, secondary1SpID, secondary2SpID
         }
-        """
+        """,
+        responses={200: "Success", 400: "Bad request", 500: "Server error"},
+    )
+    @full_ns.marshal_with(users_by_content_node_response)
+    # @cache(ttl_sec=30)
```

Comment:
interesting, not sure why this is commented out [(it always has been, since i added it lol)](https://github.com/AudiusProject/audius-protocol/blame/7685838bc6585215dd82ca9022c43442ed2a5fd7/discovery-provider/src/api/v1/users.py#L841)

prob should see if anything breaks by adding this cache, but can handle separately from this PR if you want (lets try to do as part of this immediate work tho bc theres really no reason not to cache)

Reply:
ok I added it back. seems like it's not causing any issues but I'll monitor after deploying to staging and it'll be easy to roll back

Reply:
sweet - i'm also down to explore increasing this value, since this is a background job and does not need extremely up to date data. like 5min cache prob is good? wdyt

Reply:
I think 5 minutes sounds totally reasonable. changed it to that now